sunpy/map/mapbase.py

```diff
@@ -159,7 +163,7 @@ def __init_subclass__(cls, **kwargs):
         if hasattr(cls, 'is_datasource_for'):
             cls._registry[cls] = cls.is_datasource_for
 
-    def __init__(self, data, header, plot_settings=None, **kwargs):
+    def __init__(self, data, header, cmap='gray', norm=None, plot_settings=None, **kwargs):
```

Comment:
Given conversations with @Cadair and @dstansby, we don't want to accept `cmap` and `norm` into the constructor so remove these two kwargs